### PR TITLE
Add README to elemetal-operator helm chart

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by Itxaka** [comment]
I'm wondering how this makes sense with our splitter chart push. I.E. on this repo we push to our GitHub helm repo but on the readme we mention the opensuse OCI registry.

I guess we should have a different reqdme here pointing to our GitHub resources and then add a different reqdme on the obs build?


Also, would it be possible to generate the readme with some helm plugin that takes those parameters as flags maybe? That would be pretty cool and we could generate it as part of the chart build 


Also, docker push jobs doesn't work with forks, only with branches off the main repo so we can ignora that job for now 😀

File: chart/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+```console
+helm show values oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+```
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| image.empty | string | `rancher/pause:3.1` |  |
+| image.repository | string | `rancher/elemental-operator` | Source image for elemental-operator with repository name  |
```
Comment (fgiudici) [inline chart/README.md:65]: rancher/elemental would seem like the image is on dockerhub ... instead we have:
on github:  quay.io/costoolkit/elemental-operator-ci
on OBS: registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-operator
should we put the full OBS path?

**Comment by fgiudici** [comment]
> I'm wondering how this makes sense with our splitter chart push. I.E. on this repo we push to our GitHub helm repo but on the readme we mention the opensuse OCI registry.
> 
> I guess we should have a different reqdme here pointing to our GitHub resources and then add a different reqdme on the obs build?
> 
> Also, would it be possible to generate the readme with some helm plugin that takes those parameters as flags maybe? That would be pretty cool and we could generate it as part of the chart build
> 
> Also, docker push jobs doesn't work with forks, only with branches off the main repo so we can ignora that job for now grinning

Or should we mention that the official release is on OBS (all as in the current README) and keep a separate "Dev" instructions detailing the installation from the github?

**Comment by Itxaka** [comment]
> Or should we mention that the official release is on OBS (all as in the current README) and keep a separate "Dev" instructions detailing the installation from the github?

That could also work, dev charts VS official charts, nice!

**Review by fgiudici — APPROVED** [review]
Probably we should add some more context, because we release the chart here on github, while we point to the OBS release in the README here. Anyway, I think this is a good start, we can improve this later.
LGTM, thanks @mjura !

**Pull request merged**